src/content/docs/building-with-kessel/how-to/report-resources.mdx

```diff
@@ -140,3 +140,166 @@ sent, it will miss the notification. Listening should be one of the first things
 because LISTEN/NOTIFY is a long-lived operation that can block other operations on the same connection. You should also 
 aim to minimize connections in a way that each new LISTEN does not produce a new connection, but rather reuses an 
 existing one.
+
+## Kafka Consumer Strategies
+As a service provider, you need a reliable way to replicate data changes from your systems to the Fabric. 
+A Kafka consumer offers a robust and scalable pattern to solve this problem. A consumer reads messages from Kafka topics by subscribing to topic partitions. 
+The consuming application then processes the messages to apply those changes.
+
+This guide outlines the key challenges your consumer should address and provides
+some high-level strategies for building a robust solution.
+
+We built our consumer in Go using the [confluent-kafka-go](https://github.com/confluentinc/confluent-kafka-go) client library.
+
+### Why Use a Kafka Consumer for Replication?
+When replicating data, a Kafka consumer solves the core problem of reliably processing asynchronous messages.
```

Comment:
If I was a service provider who had no experience with Kafka/Consumers etc. I'm not sure I would fully understand how/when a consumer is useful for replication. Is someone sending my consumer messages? Should I be producing messages for my consumer? We say it "solves the core problem of reliably processing async messages" but I'm not sure there are other solutions. I wonder if we should call out this as more of a necessity if someone is wanting to consume messages from a specific topic for their data replication lifecycle.